cernopendata/modules/fixtures/data/docs/delphi-getting-started/delphi-getting-started.md

```diff
@@ -0,0 +1,229 @@
+* Laboratory identifier CERN (please change that if possible)
+* year 2000 (no TPC sector 6 period)
+
+The events will be reconstructed and an extended short DST file will be created, which is ready for analysis.
+
+Created files:
+
+* simana.fadsim : detector simulation output (corresponds to raw data, not to be used directly)
+* simana.fadana : reconstructed output, full DST format (not to be used directly)
+* simana.xsdst  : short DST output for analysis
+
+In principle, analysis code can be run as well on the full DST output, however, this requires that a bunch of modules and fixes have to be rerun on top of the data, before they can be used.
+
+### Using external generators
+In this case the generator is run externally and the output is written to a file in a specific format. This can then
+be passed through the detector simulation with
+```
+runsim -VERSION va0u -LABO CERN -NRUN 1000 -EBEAM 45.625 -gext generated.lund -NEVMAX 10
+```
+Old executables for generators can be found in /cvmfs/delphi.cern.ch/mc-production/generators/pgf77_glibc2.2
+
+A source code example of DELPHI tuned Pythia can be found in the example tree.
+
+## Running an analysis job on the result
+The following script can be run interactively or submitted to a batch farm with DELPHI setup
+
+```
+#!/bin/bash
+pgm=skelana
+
+export DELLIBS=`dellib skelana dstana pxdst vfclap vdclap ux tanagra ufield bsaurus herlib trigger uhlib dstana`
+export CERNLIBS=`cernlib  genlib packlib kernlib ariadne herwig jetset74`
+echo "+OPTION VERbose" > $pgm.cra1
+echo "+USE, ${PLINAM}." >> $pgm.cra1
+cat $DELPHI_PAM/skelana.cra >> $pgm.cra1
+
+# modify
+ycmd=`which nypatchy`
+command="$ycmd - $pgm.f $pgm.cra1 $pgm.ylog $pgm.c - - ${pgm}R.f .go"
+echo "Running $command"
+eval $command
+
+# compile
+for ending in .f .F ; do
+    ls *$ending >/dev/null 2>&1
+    if [ $? -eq 0 ]; then
+	for file in *$ending  ; do
+	    $FCOMP $FFLAGS -c $file
+	done
+    fi
+done
+
+for ending in  *.c *.C *.cc ; do
+    ls *$ending >/dev/null 2>&1
+    if [ $? -eq 0 ]; then
+	for file in *ending ; do
+	    $CCOMP $CFLAGS -c $file
+	done
+    fi
+done
+
+# link
+$FCOMP $LDFLAGS *.o -o $pgm.exe $ADDLIB $DELLIBS $CERNLIBS
+
+# cleanup
+rm -f *.f *.c *.o
+
+# create input file
+echo "FILE = simana.xsdst" > ./PDLINPUT
+
+# execute
+./$pgm.exe 1>$pgm.log 2>$pgm.err
+```
+
+This simple script will:
+
+* get the sources
+* run nypatchy to create the compilable Fortran input files
+* compile the Fortran code to create an executable file
+* create a data input fiel which would read **simana.xsdst** from the local folder where the executable will be started
+* run the job
+
+## Using nicknames
+DELPHI data is organised in data sets which are identified via a nickname. When using opendata, each nickname comes with a corresponding DOI which you can quote.
+
+To analyse data, use the nicknames which you can find at http://delphiwww.cern.ch/offline/data/castor/html.
```

Comment:
Would it be interesting to make this a page on the portal, so that the documentation corner would be "standalone" in case the web site is down for a reason or another? (Thinking of long-term preservation; see e.g. the LHCb stripping lines etc that brings very detailed documentation accompanying the release and that was also scraped from the collaboration internal sources.)

Reply:
This we could do, maybe not urgent for the first iteration. I have opened an internal ticket to track this.
These pages are nested and auto-generated. We have to see what's the best way to proceed with this.